platform-operator/platformcontroller.go

```diff
@@ -138,7 +139,8 @@ func NewPlatformController(
 				// Two different versions of the same Deployment will always have different RVs.
 				return
 			} else {
-				controller.enqueueFoo(new)
+				//controller.enqueueFoo(new)
```

Comment:
Is enqueueFoo not needed?

Reply:
Yes, I misunderstood what it was doing. Fixed.